package.json

```diff
@@ -40,11 +40,11 @@
   },
   "peerDependencies": {
     "enzyme": "^3.1.0",
-    "inferno": "^4.0.4",
-    "inferno-create-element": "^4.0.4",
-    "inferno-server": "^4.0.4"
+    "inferno": "^5.0.5",
```

Comment:
If this code works with 5.x without any changes, then the peerDependencies should include 4.x That way users with Inferno 4 can continue to use it.

Please replace the versions with 4.x or 5.x:

```
"inferno": "4.x || 5.x"
```

Reply:
Good point will do.